packages/jest-bolt-receiver/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import { events } from '@slack-wrench/fixtures';
+import {
+  BlockButtonAction,
+  MessageEvent,
+  Receiver,
+  ReceiverEvent,
+  SlashCommand,
+} from '@slack/bolt';
+import { EventEmitter } from 'events';
+
+interface ButtonActionArgs {
+  action_id?: string;
+  block_id?: string;
+  value?: string;
+}
+
+export default class JestReceiver extends EventEmitter implements Receiver {
+  private send(body: Record<string, any>): ReceiverEvent {
+    const event: ReceiverEvent = {
+      body,
+      ack: jest.fn(),
+    };
+
+    if (body.response_url) {
+      event.respond = jest.fn();
+    }
+
+    this.emit('message', event);
+
+    return event;
+  }
+
+  // For things that fall under "Base Events"
+  // https://github.com/slackapi/bolt/blob/1655999346077e9521722a667414758da856ede2/src/types/events/base-events.ts
```

Comment:
why link to a specific tree/commit? why not https://github.com/slackapi/bolt/blob/master/src/types/events/base-events.ts ?